azure-pipelines.yml

```diff
@@ -160,10 +160,16 @@ jobs:
   - template: ci/checkout-template.yml
   - template: ci/install-package-dependencies.yml
   - bash: ci/scripts/check-bazel-tags.sh
-    displayName: Check Bazel Tags
+    displayName: Check Bazel Tags (Experimental)
     continueOnError: True
   - bash: ci/scripts/check-bazel-banned-rules.sh
     displayName: Check for banned rules
+  - bash:  ci/scripts/check_bazel_target_names.py
+    displayName: Check Bazel target names (Experimental)
+    continueOnError: True
+  - bash:  ci/scripts/check_bazel_test_suites.py
```

Comment:
If the order here is important, would it be worthwhile to combine these Bazel checks into a single job? Something like a "Bazel lint" job (though I'm not sure if this really qualifies as linting).

Reply:
> Can you put this check before "displayName: Check Bazel Tags (Experimental)"

Done.

> Can we also have a comment to show the order is considered ad should have that effect.

Done.

> If the order here is important, would it be worthwhile to combine these Bazel checks into a single job? Something like a "Bazel lint" job (though I'm not sure if this really qualifies as linting).

I wouldn't mind combining them into a single job, but it's just slightly trickier than I want to deal with in this PR. I figure we should unconditionally run all of the checks, then fail the job if any check failed. That means we need more control flow than `set -e ; command1 ; command 2; ...`.

WDYT about eventually moving the Bazel checks into Python and fleshing out the `BazelQuery` class as needed? If Bazel checks were just Python functions, it would be trivial to combine them into one Azure Pipelines job.